src/core/a-assets.js

```diff
@@ -83,9 +83,11 @@ registerElement('a-asset-item', {
     attachedCallback: {
       value: function () {
         var self = this;
-        var src = this.getAttribute('src');
+        var src;
 
+        src = this.getAttribute('src');
```

Comment:
In that case, note that we got tempted to initialize `embedded` even though it might not even get used if it returns early: https://github.com/aframevr/aframe/blob/master/src/core/scene/a-scene.js#L266 

Mixing declaration/logic also messes up the alphabetization sometimes if the variables depend on each other.

I am not sure where to draw the line. Perhaps only allow pure declarations and aliasing (e.g., `var el = this.el`) in the declaration block. Everything else is logic and goes after.